charts/tfy-agent/Chart.yaml

```diff
@@ -15,5 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.50-rc.2
+version: 0.2.49
 
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
```

Comment:
This is incorrect, looks like change from older main branch

appVersion was removed
next version should be `0.2.50-rc.3`